lib/query/query_recurring_donations.rb

```diff
@@ -142,8 +154,9 @@ def self.for_export_enumerable(npo_id, query, chunk_limit=15000)
   end
 
   def self.get_chunk_of_export(npo_id, query, offset=nil, limit=nil, skip_header=false )
-    root_url = query[:root_url]
+    root_url = query[:root_url] || 'https://us.commitchange.com/'
     include_stripe_customer_id = query[:include_stripe_customer_id]
+    include_last_failed_charge = query[:include_last_failed_charge] || false
```

Comment:
If the goal here is to get a boolean, I would use `!!query[:include_last_failed_charge]` to the right side.